modelina-website/src/components/playground/options/TypeScriptGeneratorOptions.tsx

```diff
@@ -161,8 +205,28 @@ class TypeScriptGeneratorOptions extends React.Component<
           </li>
         ) : null}
         {this.context?.tsModelType === 'class' ? (
-          <li>
-            <label className="flex items-center py-2 justify-between cursor-pointer">
+          <li className='flex items-center'>
+            <InfoModal text='Typesript include JsonBinPack support: ' >
+              <p>
+                It indicates whether the JsonBinPack support should be included in the generated code.
+                <br/> <br/>
+                The default value is false.
+                <br/><br/>
+                <ul className='list-disc list-inside'>
+                  <li className='list-disc'>This functionality is for the library jsonbinpack.</li>
+                  <li>This preset can ONLY be used with AsyncAPI 2.x and JSON Schema draft 4 to 7 inputs.</li>
+                  <li>
+                    This functionality has two requirements:
+                    <ol className='list-decimal list-inside'>
+                      <li>You MUST manually install the library jsonbinpack.</li>
+                      <li>You MUST also use the Generate un/marshal functions for classes</li>
+                    </ol>
+                  </li>
+                  <li>This feature allows you to convert models to a buffer, which is highly space-efficient, instead of sending pure JSON data over the wire.</li>
```

Comment:
I would move this sentence to the top to form something like:

```
It indicates whether the JsonBinPack support should be included in the generated code. This allows you to convert models to a buffer, which is highly space-efficient, instead of sending pure JSON data over the wire. 
```